kernel/enter.c

```diff
@@ -820,7 +820,8 @@ int myst_enter_kernel(myst_kernel_args_t* args)
                  args->envc,
                  args->envp,
                  NULL,
-                 NULL)) != 0)
+                 NULL,
+                 args->main_stack_size)) != 0)
```

Comment:
Is it necessary to add a new input parameter for main_stack_size? It seems that other configurations did not get specific input parameter.

Reply:
You're right. It is interesting that `myst_exec` takes many input parameters but `crt_data`, `crt_size`, `crt_reloc_data`, `crt_reloc_data` can be retrieved from global data structure `__myst_kernel_args.` So does `main_stack_size`.

Reply:
Fixed.